core/src/main/java/bio/terra/pearl/core/dao/search/EnrolleeSearchExpressionDao.java

```diff
@@ -103,12 +104,19 @@ public EnrolleeSearchExpressionResult map(ResultSet rs, StatementContext ctx) th
                     );
                 }
 
-                if (columnName.startsWith("mailing_address_")) {
+                if (columnName.startsWith("mailing_address_") && columnName.endsWith("_created_at")) {
                     enrolleeSearchExpressionResult.setMailingAddress(
                             BeanMapper.of(MailingAddress.class, "mailing_address")
                                     .map(rs, ctx)
                     );
                 }
+
+                if (columnName.startsWith("latest_kit_") && columnName.endsWith("_created_at")) {
```

Comment:
this is four `if`s in a row with extremely similar code -- let's find a way to extract a helper function